src/ParallelAlgorithms/Amr/Projectors/DefaultInitialize.hpp

```diff
@@ -0,0 +1,41 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <utility>
+
+#include "ParallelAlgorithms/Amr/Protocols/Projector.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+template <size_t Dim>
+class Element;
+template <size_t Dim>
+class Mesh;
+/// \endcond
+
+namespace amr::projectors {
+
+/// \brief Initialize/update items related to time stepping after an AMR change
+template <typename... Tags>
```

Comment:
We generally prefer typelists rather than parameter packs as template arguments because they're easier to work with. Want to switch this and expand in a specialization?